addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -1089,6 +1094,9 @@ function _process_request_for_internal_user(store, name, params) {
         ];
         store.add(this.env["mail.canned.response"].search(domain));
     }
+    if (name === "res.role") {
+        store.add(this.env["res.role"].search([["name", "ilike", params.term]]));
```

Comment:
It's best to specific which fields (in case we add other one) to correctly match python.